### PR TITLE
reformat yash.md and add some content

### PR DESCRIPTION
**Pull request by ndushay**
Taking John's PR #3 to the next level

### CODE REVIEW

File: yash.md [hunk 1]
```diff
@@ -2,7 +2,85 @@
...
+- ask Andrew
+  * APO
+  * Collection
+  * Item
+
+## cocina-models
+_SDR data model written as syntactically validatable with dry-struct, dry-types and openapi_
+- ask JCoyne or JLitt
+
+## dor-services-app
```
Comment (jcoyne) [inline yash.md:35]: Is this duplicative of https://github.com/sul-dlss/infrastructure-team/blob/gh-pages/sdr/index.md#dor-services-app-dsa ?

File: yash.md [hunk 2]
```diff
@@ -2,7 +2,85 @@
 
 _A combination index, glossary, and high level guided tour for the Stanford Digital Repository_
 
-* Access Rights for digital content
+Intended Audience: infrastructure team developers
+
+## Argo
```
Comment (jcoyne) [inline yash.md:7]: Duplicative of https://github.com/sul-dlss/infrastructure-team/blob/gh-pages/sdr/index.md#argo

File: yash.md [hunk 3]
```diff
@@ -2,7 +2,85 @@
...
+## dor-services-app
+_In an effort to isolate our code that directly depends on Fedora3, the DSA was created.  It has grown to be a sort of kitchen sink nexus of a great deal of cross-application / cross-service needs, exposed as API calls._
+
+## Workflows
+
+### Workflow Server
+_a Rails App to track the workflows associated with an object, and the state of each step and an error string for failed steps_
+- ask JCoyne
+
+## Robots
```
Comment (jcoyne) [inline yash.md:59]: can we put this in https://github.com/sul-dlss/infrastructure-team/blob/gh-pages/sdr/index.md#robots ?

File: yash.md [hunk 4]
```diff
@@ -2,7 +2,85 @@
...
+## SURI
+_Stanford University Repository Identifier service - it mints our identifiers_
+- ask JCoyne or Naomi or ...
+  * we sometimes call our identifiers druids - Digital Repository Unique IDs
+  * they have a very specific format -- see https://github.com/sul-dlss/druid-tools and/or https://consul.stanford.edu/pages/viewpage.action?title=SURI+2.0+Specification&spaceKey=chimera
+  * SURI app is a rails app: https://github.com/sul-dlss/suri-rails
+
+
+## Puppet
+_SUL DLSS uses Puppet for its lower level IT infrastructure_
```
Comment (jcoyne) [inline yash.md]: To do what?

File: yash.md [hunk 5]
```diff
@@ -2,7 +2,85 @@
...
+_robots to preserve objects being accessioned_
+- ask Naomi
+#### gis-robots
+_robots to carry out special processing required for GIS objects_
+- ask anyone as we are all equally ignorant, more or less
+#### was_robot_suite
+_robots to carry out special processing required for WAS seed and crawl objects_
+- ask Naomi or JLitt
+
+## SURI
```
Comment (jcoyne) [inline yash.md:108]: Should this be on https://github.com/sul-dlss/infrastructure-team/blob/gh-pages/sdr/index.md ?

**Review by jcoyne — COMMENTED** [review]
Too me, dumping everything in one page is not providing a lot of structure which would makes this information accessible.  Why do we describe our services on two different pages?

**Comment by ndushay** [comment]
@jcoyne THis is an intermediate step.  John will be merging this into index.md or something like it.   (You are seeing sausage making at the moment)

**Review by jmartin-sul — APPROVED** [review]

**Pull request merged**